README.md

```diff
@@ -1 +1,5 @@
 # Tari Blockchain Explorer API
+
+## Development
+
+Use `NO_CRON=1 npm run start` to stop the cronjobs from running.
```

Comment:
`NO_CRON=1 npm run dev` is a slightly better guideline, IMO.

Reply:
Agreed